README.md

```diff
@@ -17,7 +17,7 @@ Dapr is a portable, event-driven, serverless runtime for building distributed ap
 
 A client can be created as follows:
 
-```
+```rust
 extern crate async_trait;
 extern crate dapr;
```

Comment:
This doesn't have to do with your changes, but both of these `extern crate` lines are not needed

Reply:
Ok, I'll delete them then ^^